ServiceApp/src/org/droidplanner/services/android/core/gcs/follow/LocationRelay.java

```diff
@@ -0,0 +1,121 @@
+package org.droidplanner.services.android.core.gcs.follow;
+
+import android.content.Context;
+
+import com.o3dr.services.android.lib.coordinate.LatLongAlt;
+
+import org.droidplanner.services.android.core.gcs.location.Location;
+
+import timber.log.Timber;
+
+/**
+ * Created by kellys on 2/24/16.
+ */
+public class LocationRelay {
```

Comment:
@kellyschrock this class seems to replicate a lot of the functionality in the `FusedLocation`. I think it makes sense for the functionality to be here, so we should remove it from the `FusedLocation` class, and have the `FusedLocation` class call this class instead and pass the updated locations through here.